kubernetes-controllers/node-exporter-daemonset.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: node-exporter
+  labels:
+    app.kubernetes.io/name: node-exporter
+    app.kubernetes.io/version: v0.18.1
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: node-exporter
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: node-exporter
+        app.kubernetes.io/version: v0.18.1
+    spec:
+      tolerations:
+      # this toleration is to have the daemonset runnable on master nodes
+      # remove it if your masters can't run pods
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
```

Comment:
Вы правильно определили tolerations для данного случая, но данное решение не является универсальным.
Предлагаю вам попробовать сделать его таковым, а для этого подробнее ознакомиться с документацией:
https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/#concepts -> секция Note и с официальным манифестом для node exporter.

Reply:
Hi @Jasstkn, I changed tolerations options to operator: Exists with no key.  An empty key with operator Exists matches all keys, values and effects which means this will tolerate everything. So I think that the problem was solved, and I made manifest more universal

Reply:
Exactly what we need. Good work, approved!